drivers/serial/uart_npcx.c

```diff
@@ -209,6 +212,12 @@ static void uart_npcx_isr(const struct device *dev)
 {
 	struct uart_npcx_data *data = DRV_DATA(dev);
 
+	/* Refresh console expired time if got UART Rx event */
+	if (IS_ENABLED(CONFIG_UART_CONSOLE_INPUT_EXPIRED) &&
+						uart_npcx_irq_rx_ready(dev)) {
```

Comment:
line seems over-indented (should align to left-paren of if)

Reply:
Done.